config/src/test/java/com/megaease/easeagent/config/yaml/YamlReaderTest.java

```diff
@@ -42,12 +42,16 @@ public void testCompress() {
 
         yamlMap.forEach((k, v) -> {
             Assert.assertTrue(yamlProperties.containsKey(k));
-            Assert.assertEquals(v, yamlProperties.get(k));
+            if (!k.equals("name")) {
```

Comment:
I think the name's parameter, in agent.properties in test/resources, set as `test-service` would be better